app/middlewares/validateSchema.js

```diff
@@ -0,0 +1,10 @@
+const Ajv = require('ajv');
+const addFormats = require('ajv-formats')
+
+const ajv = new Ajv()
+addFormats(ajv);
+
+module.exports.validateSchema = schema => (req, res, next) => {
+  const validate = ajv.compile(schema);
+  validate(req.body) ? next() : res.status(400).send(validate.errors);
```

Comment:
here you should send a custom error to the error handler middleware through the `next` function